fdbclient/BackupAgentBase.actor.cpp

```diff
@@ -21,6 +21,7 @@
 #include <iomanip>
 #include <time.h>
 
+#include "flow/Hash3.h"
```

Comment:
why was this not needed before but is needed now?

Reply:
It used to be transitively included before - the change is that flow/FastAlloc.h no longer include flow/Hash3.h

Reply:
oh I didn't realize this. Sounds good, this is probably better